src/common/dom/apply_themes_on_element.ts

```diff
@@ -21,26 +21,33 @@ const hexToRgb = (hex: string): string | null => {
  * element: Element to apply theme on.
  * themes: HASS Theme information
  * localTheme: selected theme.
- * updateMeta: boolean if we should update the theme-color meta element.
+ * mainElement: boolean if it is the top element of the page.
  */
 export const applyThemesOnElement = (
   element,
   themes,
   localTheme,
-  updateMeta = false
+  mainElement = false
 ) => {
-  if (!element._themes) {
-    element._themes = {};
-  }
   let themeName = themes.default_theme;
   if (localTheme === "default" || (localTheme && themes.themes[localTheme])) {
     themeName = localTheme;
   }
+  if (!element._themes) {
+    if (themeName === "default" || (!localTheme && !mainElement)) {
```

Comment:
This will never be `default` because you override it in the if-statement above it.

Reply:
No, if `localTheme === "default"` => `themeName = localTheme` => `themeName === "default"`